### ISSUE TITLE
qt applications don't work fine on gnome(wayland) after using TheShell

### ISSUE THREAD

**Issue by parhambarzegar** [vicr123/theshell#39]

**Comment by vicr123** [comment]
Can I get some more details please? :)

**Comment by parhambarzegar** [comment]
sure: https://postimg.cc/gallery/idw0nbvy/
environment variables: https://beepaste.io/paste/view/DK2Ig1
i'm also not able to move that window.

**Comment by vicr123** [comment]
Does that happen in all Qt applications? Does it happen in Weston? :) 

Also, what OS are you using?

**Comment by parhambarzegar** [comment]
yes. almost doesn't, tough they're window frames are still like in gnome, in gnome they work fine by setting QT_QPA_PLATFORM=xcb

archlinux

**Comment by vicr123** [comment]
I'm not quite sure why theShell would get GNOME to set the QT_QPA_PLATFORM variable to something else :thinking: 

What is it by default? Over on my machine it's not set so I assume by default it's set to xcb

**Comment by parhambarzegar** [comment]
Due to my env variables gnome doesn't set any variable related to qt
it is probably xcb by default

It still might be because of plasma😬, I used that too.
isn't there any config file related to qt? except in home, i've tried changing my home.

**Comment by parhambarzegar** [comment]
and what about kgpg? it autostarts on gnome.
ah this one is probably because of kde.

**Comment by vicr123** [comment]
I don't think it's got anything to do with KGpg but you can try turning it off and it might fix it :)

Unfortunately I don't really have much experience with Wayland :(

**Comment by parhambarzegar** [comment]
I'm sure that it hasn't, that's another issue
thanks anyway :)
